index.html

```diff
@@ -151,6 +151,27 @@ <h3>Skills</h3>
       </li>
     </ul>
     </section>
+    <section id="form-container">
+      <header>
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h2>
+      </header>
+      <form id="form-element" action="https://formspree.io/f/meqdoagj" method="POST">
+        <input class="input-field" type="text" name="name" maxlength="30" placeholder="Full name" required>
+        <input class="input-field" type="email" name="email" placeholder="Email address" required>
+        <textarea id="text-area" class="input-field" name="message" maxlength="500" placeholder="Enter text here" required></textarea>
+        <button id="form-button" type="submit">Get in touch</button>
+      </form>
+      <ul class="socials">
+        <li><a href="#"><img src="./pics/github.png" alt="github logo"></a></li>
+        <li><a href="#"><img src="./pics/linkedin.png" alt="linkedin logo"></a></li>
+        <li><a href="#"><img src="./pics/Enabled.png" alt="enabled"></a></li>
+        <li><a href="#"><img src="./pics/twitter.png" alt="twitter"></a></li>
+        <li><a href="#"><img src="./pics/medium.png" alt="medium"></a></li>
+      </ul>
+      <div id="email">edoziechukwuma@gmail.com</div>
```

Comment:
- [ ] [Optional] [Highly Recommended] Please add these social links and email inside a `<footer>` tag for semantics purposes.
### ⚠️I marked this as optional because the last reviewer forgot to mention it so please consider doing it⚠️